sympy/functions/elementary/exponential.py

```diff
@@ -436,11 +436,15 @@ def _eval_nseries(self, x, n, logx):
         if arg0 in [-oo, oo]:
             return self
         t = Dummy("t")
-        exp_series = exp(t)._taylor(t, n)
+        nterms = n + 2
+        cf = Order(arg.as_leading_term(x), x).getn()
+        if cf != 0:
+            nterms = int(nterms / cf)
```

Comment:
This should be `(n/cf).ceiling()` as in `Function._eval_nseries()`.

Reply:
oops.. sorry :o